jsonargparse_tests/test_core.py

```diff
@@ -167,6 +167,23 @@ def test_default_env(self):
             self.assertFalse(parser.default_env)
 
 
+    def test_env_prefix(self):
+        for env_prefix in [None, True]:
+            parser = ArgumentParser(env_prefix=env_prefix, default_env=True)
+            parser.add_argument("--test_arg", type=str, required=True, help="Test argument")
+            with self.assertRaises(SystemExit):
```

Comment:
Please make sure that parsing error messages are not seen in the test logs. Options are:
- Instantiate `ArgumentParser` with `error_handler=None` and assert `ParserError` instead of `SystemExit`
- In this line add one more context manager `, suppress_stderr():`

Reply:
Got it, done